src/ops/i2c.h

```diff
@@ -26,4 +26,35 @@ extern const tele_op_t op_IIBB1;
 extern const tele_op_t op_IIBB2;
 extern const tele_op_t op_IIBB3;
 
+extern void i2c_get_0(command_state_t *cs, uint8_t addr, uint8_t cmd);
```

Comment:
are these generic i2c helpers? i would put them into a separate file under src - this one is for the generic i2c ops.